specs/wp-signup-spec.js

```diff
@@ -1590,4 +1592,75 @@ describe( `[${ host }] Sign Up  (${ screenSize }, ${ locale })`, function() {
 			await NoSitesComponent.Expect( driver );
 		} );
 	} );
+
+	describe.only( 'Import a site while signing up @parallel', function() {
+		const siteURL = '';
+		const userName = dataHelper.getNewBlogName();
+		const emailAddress = dataHelper.getEmailAddress( userName, signupInboxId );
+
+		before( async function() {
+			return await driverManager.ensureNotLoggedIn( driver );
+		} );
+
+		step( 'Can visit import in signup page', async function() {
+			await StartPage.Visit( driver, StartPage.getStartURL( { culture: locale, flow: 'import' } ) );
+		} );
+
+		step( 'Can then enter the url of a site to import', async function() {
+			const importFromURLPage = await ImportFromURLPage.Expect( driver );
+
+			await importFromURLPage.enterURL( siteURL );
+			return await importFromURLPage.submitForm();
+		} );
+
+		step( 'Can then enter account details and continue', async function() {
+			const createYourAccountPage = await CreateYourAccountPage.Expect( driver );
+
+			return await createYourAccountPage.enterAccountDetailsAndSubmit(
+				emailAddress,
+				userName,
+				passwordForTestAccounts
+			);
+		} );
+
+		step( 'Can activate my account from an email and go to the importers page', async function() {
+			const emailClient = new EmailClient( signupInboxId );
+			const validator = emails => emails.find( email => email.subject.includes( 'Activate' ) );
+			let emails = await emailClient.pollEmailsByRecipient( emailAddress, validator );
+			assert.strictEqual(
+				emails.length,
+				1,
+				'The number of newly registered emails is not equal to 1 (activation)'
+			);
+			const activationLink = emails[ 0 ].html.links[ 0 ].href;
+			assert( activationLink !== undefined, 'Could not locate the activation link email link' );
+			await driver.get( activationLink );
+		} );
+
+		step( 'Can then see preview of site to be imported', async function() {
```

Comment:
Something to keep in mind, I've seen the `is-site-importable` call fail (super rare I'd imagine, but it happens), so we need to make sure the assertion in this test will be true even if an API call fails (or we'll need to mock it).